brambling/static/brambling/sass/modules/_event-listing.sass

```diff
@@ -1,6 +1,6 @@
 .event-listing-image
 	width: 100%
-	padding-top: 27%
+	padding-top: (300 / 940) * 100%
```

Comment:
Can we document where this calculation comes from? Also sass has a [`percentage`](http://sass-lang.com/documentation/Sass/Script/Functions.html#percentage-instance_method) function for what I think you're trying to do with the `* 100%` deal. :)